From 99ac111c2604a47d865bd4d7996be8cfaf2af146 Mon Sep 17 00:00:00 2001
From: Anatoly Parshintsev <anatoly.parshintsev@intel.com>
Date: Wed, 16 Mar 2022 09:37:03 +0000
Subject: [PATCH] add PVCXT_A0 platfrom

add PVCXT_A0 platform and refactor the list initialization to
simplify maintenance
---
 .../include/llvm/GenXIntrinsics/Intrinsics.py | 25 ++++++++++++++++---
 1 file changed, 22 insertions(+), 3 deletions(-)

diff --git a/GenXIntrinsics/include/llvm/GenXIntrinsics/Intrinsics.py b/GenXIntrinsics/include/llvm/GenXIntrinsics/Intrinsics.py
index 3e08425..92fc783 100755
--- a/GenXIntrinsics/include/llvm/GenXIntrinsics/Intrinsics.py
+++ b/GenXIntrinsics/include/llvm/GenXIntrinsics/Intrinsics.py
@@ -78,9 +78,28 @@ attribute_map = {
 
 # order does really matter.
 # It is used to define ordering between the respected platforms
-platform_list = ["HSW", "BDW", "CHV", "SKL", "BXT", "KBL",
-        "GLK", "CNL", "ICL", "ICLLP", "TGLLP", "RKL", "DG1",
-        "ADLP", "ADLS", "XEHP", "DG2", "PVC", "PVCXT"]
+platform_list = [
+    "HSW",
+    "BDW",
+    "CHV",
+    "SKL",
+    "BXT",
+    "KBL",
+    "GLK",
+    "CNL",
+    "ICL",
+    "ICLLP",
+    "TGLLP",
+    "RKL",
+    "DG1",
+    "ADLP",
+    "ADLS",
+    "XEHP",
+    "DG2",
+    "PVC",
+    "PVCXT_A0",
+    "PVCXT",
+]
 
 def getAttributeList(Attrs):
     """
-- 
2.34.1

